### PR TITLE
seeding

### PR DESCRIPTION
**Pull request by egppp**
guys, have a look at the seeding

### CODE REVIEW

**Comment by PierrevHL** [comment]
wow looks great :)

**Comment by egppp** [comment]
show me that index....

File: app/models/booking.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class Booking < ApplicationRecord
   belongs_to :profile
   belongs_to :user
+  validates :start_date, :end_date, presence: true
```
Comment (PierrevHL) [inline app/models/booking.rb:4]: uhhi dont think this istrue

**Review by PierrevHL — APPROVED** [review]
good

**Pull request merged**